projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts

```diff
@@ -734,6 +734,7 @@ export const IDENTIFIERS_TO_REPLACE: ReplacementIdentifierMulti[] = [
         to: {
             name: 'TuiDialog',
             moduleSpecifier: '@taiga-ui/core',
+            spreadInModule: true,
```

Comment:
This is not going to be needed once we fix `TuiDialog`. It will not be an array.

Reply:
I removed from dialog. But we have many constants like this besides dialogue, I will add them later in separate pr